core/src/test/scala/integration/kafka/api/GroupAuthorizerIntegrationTest.scala

```diff
@@ -65,7 +65,7 @@ class GroupAuthorizerIntegrationTest extends BaseRequestTest {
   def clientPrincipal: KafkaPrincipal = ClientPrincipal
 
   override def brokerPropertyOverrides(properties: Properties): Unit = {
-    properties.put(KafkaConfig.AuthorizerClassNameProp, classOf[SimpleAclAuthorizer].getName)
+    properties.put(KafkaConfig.AuthorizerClassNameProp, classOf[AclAuthorizer].getName)
```

Comment:
Can we use ` @nowarn("cat=deprecation")` here? same for other classes and other PR changes